.gitignore

```diff
@@ -4,3 +4,5 @@ sqls/**/*.sql*
 
 # Ignore the docker-compose.override.yml file
 docker-compose.override.*
+*.env
```

Comment:
@chinzou , I believe we should not gitignore this file, unless I have missed something.
The .env file is also here to provide default values, and documentation of the possibles vars to set.

Reply:
@rbuisson I'm trying to ignore every `[name].env`  except `.env` I usually setup different configs (eg: `hsc.env`, `easydoc.hsc`)

The next line 

> !.env

 is to make sure that the default file remain.